docs/install.rst

```diff
@@ -412,12 +413,16 @@ dependencies, including:
 * `numpydoc <https://numpydoc.readthedocs.io>`_ - an extension to parse
   docstrings in NumPyDoc format
 * `pillow <https://pillow.readthedocs.io>`_ - used in one of the examples
+* `Graphviz <http://www.graphviz.org>`_ - generate inheritance graphs (ONLY
```

Comment:
"ONLY" might be too strong a word. Nothing stops you from installing it natively into your system outside of `conda`. See http://www.graphviz.org/download/

Reply:
agree, maybe just say "not a python package, you need to install it with your preferred system package manager system" or similar. Even if one uses conda for python packages, they may want to homebrew this one, etc.

Reply:
Agreed, replacing with "generate inheritance graphs (available as a conda package or a system install but not in pip)".